packages/css/baseline/ds-reset.css

```diff
@@ -2,13 +2,16 @@
 
 /* https://wiki.csswg.org/ideas/mistakes */
 
+/* TODO: Can this be moved to the related components? */
 [class^='ds-'],
 [class^='ds-']::before,
 [class^='ds-']::after {
   box-sizing: border-box;
 }
```

Comment:
This is relevant for all components, it's the same as doing 
```css
* { box-sizing: border-box } 
```
So I think this is fine to have in this file 🤔

Reply:
Not if moving to CSS modules or allowing people to extend our classes like: 
```
.my-fancy-button {
  composes: button from '@digdir/designsystemet/button.module.css'; // To extend using pure CSS
  …
}
```
Also, this targets only if elements class names _starts_ with `ds-` so `class="ds-button my-button"` works, but `class="my-button ds-button"` does not. I think it is safer and more explicit to just add `box-sizing: border-box` into the relevant classes in the relevant files :)

Reply:
Ah yeah, but this should be `class~=` 😄